lib/handlers/HeadHandler.js

```diff
@@ -11,7 +11,7 @@ class HeadHandler extends BaseHandler {
      * @return {function}
      */
     send(req, res) {
-        const re = new RegExp('\\' + this.store.path + '\\/(\\w+)\/?'); // eslint-disable-line prefer-template
+        const re = new RegExp('\\' + this.store.path + '\\/([\\w\-]+)\/?'); // eslint-disable-line prefer-template
```

Comment:
This was added just for that example hashing function from our convo on the issue, right? Maybe it should just match the non-whitespace (`\S`) rather than any alphanumeric character.